.gitignore

```diff
@@ -73,7 +73,6 @@ packages/
 /src/rendering/**/.env.local
 /src/rendering/**/.env.*.local
 /src/rendering/**/*.log*
-/src/rendering/.vercel
```

Comment:
This same pattern is already on line [69](https://github.com/sitecorelabs/xmcloud-foundation-head-dev/pull/155/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L69)